src/ar_util.erl

```diff
@@ -36,6 +36,22 @@ pick_random(List, N) ->
 pick_random(Xs) ->
 	lists:nth(rand:uniform(length(Xs)), Xs).
 
+%% @doc Pick a random element, on a `1/(1+rank)` distribution.
```

Comment:
I would clarify here ".. where rank 1 is assigned to the best peer, rank 2 to the second best peer, etc".